app/src/main/java/com/asfoundation/wallet/permissions/request/view/CreateWalletFragment.kt

```diff
@@ -19,9 +19,7 @@ import javax.inject.Inject
 
 class CreateWalletFragment : DaggerFragment(), CreateWalletView {
```

Comment:
- [ ] getCancelClick can me converted to an expression body
- [ ] getOnCreateWalletClickcan me converted to an expression body